Resources/Prototypes/_NF/PointsOfInterest/trademall.yml

```diff
@@ -0,0 +1,35 @@
+# Author Info
+# GitHub: Cheackraze, PeccNeck
+# Discord: Checkraze, mariusdemeier, trucy_deuxy
+
+# Maintainer Info
+# GitHub: ???
+# Discord: ???
+
+# Notes:
+# Basic Trade Outpost buying point for economy and great wealth
+- type: pointOfInterest
+  id: TradeMall
+  name: Trade Mall
+  rangeMin: 1500
+  rangeMax: 2500
+  IFFColor: "#37C837"
+  spawnGroup: MarketStation
+  gridPath: /Maps/_NF/POI/trademall.yml
+  busStop: true
```

Comment:
Note some of the changes on this yml are not in the comp anymore, need to update